core/src/lib.rs

```diff
@@ -61,6 +61,7 @@
 pub mod hlist;
 pub mod generic;
 pub mod labelled;
+mod tuples;
```

Comment:
Is it ok for this to not be `pub` ? I'm not sure if the implementations are useable outside this crate as is.

Reply:
Iirc, implementations belong to the crate and the trait belongs to a module. So it should be fine...
You can verify this by checking `cargo doc --open`.

Reply:
Oh, interesting. Didn't know that! Thanks, verified it locally. Should be fine.